README.md

```diff
@@ -23,6 +23,8 @@ Now you can run the script by typing (You will need netcat installed on the comp
 
     ./init_and_wait_for_db.sh
 
+**You will need to modify config.example.py with the correct database parameters.**
```

Comment:
Hmmmm... Maybe I screwed up something, but I thought I had problems with this. Nonetheless, I will look into further soon and get back to you. Thanks @Sheeo!